scout/server/blueprints/cases/views.py

```diff
@@ -873,6 +873,18 @@ def cnv_report(institute_id, case_name):
     return send_from_directory(out_dir, filename)
 
 
+@cases_bp.route("/<institute_id>/<case_name>/gene-fusion-report/<report_type>")
+def gene_fusion_report(institute_id, case_name, report_type):
+    """Download gene fusion report"""
+    _, case_obj = institute_and_case(store, institute_id, case_name)
+    report_path = case_obj.get(report_type)
```

Comment:
This is not really that dangerous, I think - as it is sort of limited to what is on the case_obj. And we have not really gone over everything like it, but since Im reading. I suppose you could if you really, really wanted add a path into like the synopsis or tissue type fields, then retrieve it by asking for report_type="synopsis" or such. Perhaps limit it to the valid keys, eg the report ones from constants?

Reply:
Especially true if we start generalizing the available reports. Very good suggestion, I'll fix it!